db_schema/queries/v1/rawApplePodcastCountries.sql

```diff
@@ -0,0 +1,6 @@
+SELECT aed_play_top_countries 
```

Comment:
`rawApplePodcastCountries.sql` is a bit misleading. I would also recommend to avoid `raw` in endpoint names as it is difficult to distinguish in the future what raw means. isn't it just `applePodcastTopCountries`?